src/components/KSelect/KSelect.vue

```diff
@@ -101,68 +101,70 @@
           </Transition>
         </div>
         <template #content>
-          <div
-            v-if="enableFiltering && loading"
-            class="select-loading"
-            data-propagate-clicks="false"
-            data-testid="select-loading"
-          >
-            <slot name="loading">
-              <ProgressIcon class="loading-icon" />
-            </slot>
-          </div>
-          <div
-            v-else
-            class="select-items-container"
-            data-propagate-clicks="false"
-          >
-            <KSelectItems
-              :items="filteredItems"
-              @selected="handleItemSelect"
+          <div :aria-live="enableFiltering ? 'polite' : 'off'">
```

Comment:
Note for PR review: the change here is adding a wrapper div with `aria-live` attribute